java/arcs/core/entity/EntityBase.kt

```diff
@@ -134,6 +134,9 @@ open class EntityBase(
     /** Returns the [FieldType] for the given singleton field, or null if it does not exist. */
     private fun getSingletonTypeOrNull(field: String) = schema.fields.singletons[field]
 
+    /** Returns true if the singleton has the Field. */
```

Comment:
This comment seems wrong. Shouldn't it be: "Returns true if the entity has the given singleton field"? (i.e. it's the field that is a singleton, not the entity)